routing/routing.go

```diff
@@ -369,8 +386,17 @@ func (r *Routing) startReceivingUpdates(o Options) {
 						r.firstLoadSignaled = true
 					}
 				}
+
+				close(goodEndpointerQuit)
+				timer.Stop()
+				goodEndpointerQuit = make(chan struct{})
+				timer = time.NewTicker(500 * time.Millisecond)
+				go GoodEndpointerGoroutine(rt.routes, goodEndpointerQuit, timer.C)
```

Comment:
Do we need to run a goroutine for each table update or maybe we can have one routine running for the whole lifetime of the datasource?

Reply:
If we create a single long-running routine it could also reuse r.quit channel.